contracts/EasyAuction.sol

```diff
@@ -476,13 +479,23 @@ contract EasyAuction is Ownable {
         if (auctionData[auctionId].minFundingThreshold > currentBidSum) {
             auctionData[auctionId].minFundingThresholdNotReached = true;
         }
-        processFeesAndAuctioneerFunds(auctionId, fillVolumeOfAuctioneerOrder);
+        processFeesAndAuctioneerFunds(
+            auctionId,
+            fillVolumeOfAuctioneerOrder,
+            auctioneerId,
+            fullAuctionedAmount
+        );
         emit AuctionCleared(
             auctionId,
             fillVolumeOfAuctioneerOrder,
             uint96(currentBidSum),
             clearingOrder
         );
+        // Gas refunds
+        auctionData[auctionId].initialAuctionOrder = bytes32(0);
+        auctionData[auctionId].interimOrder = bytes32(0);
+        auctionData[auctionId].interimSumBidAmount = uint256(0);
+        auctionData[auctionId].minimumBiddingAmountPerOrder = uint256(0);
```

Comment:
We could keep `initialAuctionOrder` and `minimumBiddingAmountPerOrder` to store an easily accessible historical record of all auctions ever made. I don't think the auctioneer minds much about the small gas savings in the transaction that's getting the profits out of the contract.

Reply:
It's all stored via logs :) 

I think it's nice to be a good citizen and clean it up